devel/ci/integration/bodhi/fedora-messaging.toml

```
@@ -0,0 +1,43 @@
+amqp_url = "amqp://rabbitmq/"
+callback = "bodhi.server.consumers:messaging_callback"
```

```suggestion
callback = "bodhi.server.consumers:Consumer"
```
This was changed for bodhi 4.0